test/server/index.js

```diff
@@ -275,6 +275,23 @@ app.post(
     }
 );
 
+app.post(
```

Comment:
Perhaps you need to edit `featureFlags` in the server since this server is run with older versions of the node SDK as well. But this is only needed if the new version of supertokens-website is still compatible with older FDIs